### PR TITLE
Remove Grafana, Dashing, Freeboard (awesome-sysadmin duplicates) #1036

### PR DESCRIPTION
**Pull request by cpdevelops**
Fixed issue #1035 by @nodiscc 

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language```
- [x] Your additions are ordered alphabetically.
- [x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -877,14 +878,12 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
   * [tastebin](https://github.com/andreineculau/tastebin) - Updated version of hastebin plus additional features. `Apache` `Nodejs`
 
 
-## Personal Dashboards
+## Personal Dashboards 
+See [here](https://github.com/n1trux/awesome-sysadmin#metric--metric-collection)
```
Comment (nodiscc) [inline README.md]: Please just use 

`See https://github.com/n1trux/awesome-sysadmin#metric--metric-collection`

for consistency with other similar links.

Comment (cpdevelops) [inline README.md]: Ok, got it. Changing now.

**Comment by Kickball** [comment]
I'll go ahead and merge this, thanks for the changes, but I am a bit confused about the issue you linked @cpdevelops, did you mean that one?

**Comment by cpdevelops** [comment]
Oh sorry @Kickball, was one number off. Meant #1036

**Pull request merged**